bucket/figurine.json

```
@@ -0,0 +1,14 @@
+{
+    "homepage": "https://github.com/arsham/figurine",
+    "description": "Print your name in style",
```

https://github.com/arsham/rainbow#see-also

```suggestion
    "description": "Print any text using ASCII art in FIGlet.",
```